fundingmanager.go

```diff
@@ -1024,11 +1018,14 @@ type chanAnnouncement struct {
 // identity pub keys of both parties to the channel, and the second segment is
 // authenticated only by us and contains our directional routing policy for the
 // channel.
-func newChanAnnouncement(localIdentity, remotePub *btcec.PublicKey,
-	channel *lnwallet.LightningChannel, chanID lnwire.ShortChannelID,
-	localProof, remoteProof *channelProof) *chanAnnouncement {
-
-	// The unconditional section of the announcement is the ChannelID
+func (f *fundingManager) newChanAnnouncement(
+	localPubKey, remotePubKey *btcec.PublicKey,
+	localFundingKey, remoteFundingKey *btcec.PublicKey,
```

Comment:
Please revert this unorthodox (for this project) indentation/wrapping style. Thanks!